tests/std/tests/P1206R7_ranges_to_mappish/test.cpp

```diff
@@ -156,13 +156,13 @@ struct mappish_instantiator {
             assert(c5.get_allocator().state == 13);
             assert(ranges::is_permutation(c5, expected, any_pair_eq));
         }
-#ifndef __EDG__ // TRANSITION, VSO-2208356
+#if defined(_MSVC_INTERNAL_TESTING) || !defined(__EDG__) // TRANSITION, VS 17.12p3
         {
             std::same_as<T> auto c6 = R{some_pairs} | ranges::to<T>(Alloc{13});
             assert(c6.get_allocator().state == 13);
             assert(ranges::is_permutation(c6, expected, any_pair_eq));
         }
-#endif // ^^^ no workaround ^^^
+#endif

```

Comment:
We should restore these "no workaround" comments so we know to remove the conditional, and not its contents, then VS 17.12p3 ships. I'll do so in a followup, this needs to match the internal PR which has already merged.

Reply:
Done in #4945.